### ISSUE TITLE
Skip to main content link not visible when in focus

### ISSUE THREAD

**Issue by michael-milette** [trema-tech/moodle-theme_trema#54]
When a keyboard only user is tabbing through the links in the page, the very first link should be the "Skip to main content" link. 

![image](https://user-images.githubusercontent.com/3808579/137516147-c1cab84e-e383-4a81-a203-12c53340f6bb.png)

However, because the page header has a higher z-index than Boost, it does not show up.

A pull request should be coming in a few minutes to address this accessibility issue.

Best regards,

Michael Milette

**Comment by rmady** [comment]
Thanks a lot @michael-milette!

I'll update version.php, so we will not have the purge caches issue anymore.
